operators/admin/olm-restricted-networks.adoc

```diff
@@ -5,26 +5,24 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-For {product-title} clusters installed on restricted networks, also known as
-disconnected clusters, Operator Lifecycle Manager (OLM) by default cannot access
-the Red Hat-provided OperatorHub sources hosted on Quay.io because they require
-full Internet connectivity. However, cluster administrators can disable those
-remote sources, create local mirrors of that same content, and configure OLM to
-install and manage Operators from the local sources instead.
+For {product-title} clusters installed on restricted networks, also known as disconnected clusters, Operator Lifecycle Manager (OLM) by default cannot access the Red Hat-provided OperatorHub sources hosted remotely on Quay.io because those remote sources require full Internet connectivity. However, as a cluster administrator you can still enable your cluster to use OLM in a restricted network if you have a workstation with full Internet access, which is only used to prepare the mirrored content.
+
+This guide describes the following process required to enable OLM in restricted networks:
+
+* Disable the default remote OperatorHub sources for OLM.
+* Use a workstation with full Internet access to create local mirrors of the OperatorHub content.
+* Configure OLM to install and manage Operators from the local sources instead.
```

Comment:
It might be helpful to add "...from the local sources instead _of the default remote sources_."